src/Builders/CustomerBuilder.php

```diff
@@ -36,4 +36,21 @@ protected function parseResponse( $response )
 
         return $items;
     }
+
+    /**
+     * @param int $id
+     *
+     * @return KgBot\Billy\Models\Customer
+     */
+    public function getWithID( $id ) 
```

Comment:
isn't this the same as `find` in the abstract class or the builder class this one is extending?